bitnami/wordpress/Chart.yaml

```diff
@@ -35,4 +35,5 @@ name: wordpress
 sources:
   - https://github.com/bitnami/bitnami-docker-wordpress
   - https://wordpress.org/
-version: 10.11.0
+version: 10.11.01
+
```

Comment:
Could you remove the last empty line?

Reply:
Sure, I also updated the version to `10.11.2`, your current version is `10.11.1`